assets/quick-order-list.js

```diff
@@ -14,16 +14,15 @@ customElements.define('quick-order-list-remove-button', QuickOrderListRemoveButt
 class QuickOrderListRemoveAllButton extends HTMLElement {
   constructor() {
     super();
-    const allVariants = Array.from(document.querySelectorAll('[data-variant-id]'));
+    const allVariants = Array.from(document.querySelectorAll('[data-quantity-variant-id]'));
```

Comment:
Previously we selected all elements on a page with `data-variant-id` and in many cases we built an array out of unnecessary elements such as <price-per-item>, pickup-availability etc while we only need to know about variants inside the Quick Order List. 

Additionally, in combination with my current logic this would break the Remove All button.